service/src/test/java/uk/gov/hmcts/reform/fpl/validation/validators/HasEPOAddressValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.fpl.validation.validators;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.enums.OrderType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+import uk.gov.hmcts.reform.fpl.model.Orders;
+import uk.gov.hmcts.reform.fpl.service.FeatureToggleService;
+import uk.gov.hmcts.reform.fpl.validation.AbstractValidationTest;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.EPOType.PREVENT_REMOVAL;
+import static uk.gov.hmcts.reform.fpl.enums.EPOType.REMOVE_TO_ACCOMMODATION;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, FeatureToggleService.class})
+class HasEPOAddressValidatorTest extends AbstractValidationTest {
+
+    @MockBean
+    private FeatureToggleService featureToggleService;
```

Comment:
If you don't want to have to define a redundant feature toggle service one way of doing it would be to make an interface with the required methods that has the validation annotation on it, the orders object can then implment that interface and the validator could validate against that interface. The test can then create a new object with implementing that interface, and then you would only be testing against that annotation and not having to accomodate for all the others on the object. Thinking about it, this might be too much work for just removing a redundant bean but food for thought

Reply:
really good idea, might be a bit too much work considering we will be removing this dependency shortly as it's a toggle. will leave for now as may require a fair bit of time to get it implemented.